templates/problem/problem.html

```diff
@@ -126,6 +131,21 @@ <h2 style="color:#393630; display: inline-block">{{ title }}</h2>
     {% endif %}
     <div><a href="{{ url('chronological_submissions', problem.code) }}">{{ _('All submissions') }}</a></div>
     <div><a href="{{ url('ranked_submissions', problem.code) }}">{{ _('Best submissions') }}</a></div>
+
+    {% if vote_perm.can_view() %}
+        <hr style="padding-top: 0.1em">
+        {% if vote_perm.can_vote() %}
+            <div><a href="#" data-featherlight="{{ url('problem_vote', problem.code) }}" id="vote-button">
+                {%- if vote is none -%}
+                    {{ _('Vote on problem points') }}
+                {%- else -%}
+                    {{ _('Change vote (%(points)d)', points=vote.points) }}
```

Comment:
Maybe reword to

```suggestion
                    {{ _('Edit points vote (%(points)d)', points=vote.points) }}
```

?